spec/services/FetchMoviesSpec.php

```diff
@@ -6,24 +6,23 @@
 
 describe(FetchMovies::class, function () {
 
-    describe("->getMovies()", function () {
+    describe("->getLatestMovie()", function () {
```

Comment:
this test does not assert anything. at some point you need to call the new method